phpmyfaq/admin/assets/src/content/stickyfaqs.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Stuff for ordering sticky faqs customly
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * @package   phpMyFAQ
+ * @author    Jan Harms <model_railroader@gmx-topmail.de>
+ * @copyright 2014-2023 phpMyFAQ Team
+ * @license   https://www.mozilla.org/MPL/2.0/ Mozilla Public License Version 2.0
+ * @link      https://www.phpmyfaq.de
+ * @since     2023-12-27
+ */
+
+import Sortable from 'sortablejs';
+import { addElement } from '../../../../assets/src/utils';
+
+export const handleStickyFaqs = () => {
+    const stickyFAQs = document.querySelector('#stickyFAQs');
+    if (stickyFAQs) {
+        Sortable.create(stickyFAQs, {
+            animation: 100,
+            draggable: '.list-group-item',
+            handle: '.list-group-item',
+            sort: true,
+            filter: '.sortable-disabled',
+            dataIdAttr: 'data-pmf-faqid',
+            onEnd: function (event) {
+                const currentOrder = Array.from(event.from.children).map(function (item) {
+                    return item.getAttribute('data-pmf-faqid');
+                });
+                saveStatus(currentOrder);
+            }
+        });
+    }
+};
+
+const saveStatus = (currentOrder) => {
+    const stickyFAQs = document.querySelector('#stickyFAQs');
+    const card = document.querySelector('#mainCardStickyFAQs');
+    const successAlert = document.querySelector('#successAlert');
+    if (successAlert) {
+        successAlert.remove();
+    }
+    fetch('./api/sticky/order', {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json, text/plain, */*',
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            faqIds: currentOrder
+        })
+    })
+            .then(async (response) => {
```

Comment:
You can use async/await as well, I refactored already a bunch of methods. It's more readable than using then()